client/src/components/ProductDetail/ProductDetail.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+import ShopListItem from '../ShopListItem/ShopListItem'
+
+export default function ProductDetail(props) {
```

Comment:
You could do destructuring assignment here: `ProductDetail({location}) {`
and then later:
```js
 // maybe call this location.product instead of location.props
const { brand, shoe, styleId, media, shops } = location.props
```